doc/api/v0.yaml

```diff
@@ -266,6 +277,11 @@ components:
           type: integer
           format: int32
           description: The fund ID this vote plan belongs to.
+        voting_token:
```

Comment:
shouldn't this be the `VoterGroup` or the group id?

Reply:
This is a direct representation of on-chain data AFAIK, could be useful if we want to be close to blockchain data (i.e. this endpoint could be served by a node as well if needed).

Reply:
> This is a direct representation of on-chain data AFAIK, could be useful if we want to be close to blockchain data (i.e. this endpoint could be served by a node as well if needed).

We have the fund_id there though, which I think doesn't have an on chain representation (although maybe I'm forgetting something here). Also, I only see this structure used in the `v0/fund/{id}` endpoint. It doesn't make much difference though, but considering that we don't have a way to map `voting token -> group` having the information there could be useful.

Reply:
> don't have a way to map voting token -> group having the information there could be useful.
We do, it's the reverse of groups -> voting tokens in the `/fund` endpoint (assuming it's bijective)

I'm slightly in favor of having the voting token here for two reasons:
* we don't have this information anywhere else
* it's "closer" to the blockchain
but having voter group could also work and maybe it's easier for the app